query-request.hh

```diff
@@ -373,7 +376,7 @@ struct forward_request {
         aggregate
     };
     struct aggregation_info {
-        cql3::functions::function_name name;
+        db::functions::function_name name;
```

Comment:
Shouldn't this happen in the patch that changed the namespace?

Reply:
Moved